packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -438,6 +438,10 @@ const AccountSwitcher = props => {
         return props.is_dxtrade_allowed;
     };
 
+    const has_reset_balance = account => {
+        return account.is_virtual && account.balance < 10000;
```

Comment:
@hamid-deriv Just a suggestion, Would be better to extract this magic number into a named const variable for better readability.